<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>520</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    QPushButton {
        background-color: #1f77b4;
        color: white;
	border-radius: 5px;
    }</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>520</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    QPushButton {
        background-color: #1f77b4;
        color: white;
	border-radius: 5px;
    }</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="monitorConfigTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>931</width>
     <height>471</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Manhole ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Monitor Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Link</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Comment</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
